### PR TITLE
Create Join_on_labels.yaml

### PR DESCRIPTION
**Pull request by Olli73773**

### CODE REVIEW

File: working-groups/query-standardization/use-cases/common/single_type/metrics/Join_on_labels.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+title: Join metrics on label strings
+
+description: >
+  A common operation operation involves combining dimensions of two or more series to allow 
+  filtering addition dimensions.
```
Comment (manolama) [inline working-groups/query-standardization/use-cases/common/single_type/metrics/Join_on_labels.yaml:5]: `s/addition/additional/`

File: working-groups/query-standardization/use-cases/common/single_type/metrics/Join_on_labels.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+inputs:
+  - metric series
+
+uses:
+  - type: Alerting
+    description: >
+      Add additional the abillity to filter on additional dimensions or get addtional demension
```
Comment (manolama) [inline working-groups/query-standardization/use-cases/common/single_type/metrics/Join_on_labels.yaml:13]: Add ~~additional~~ the

**Review by manolama — CHANGES_REQUESTED** [review]
Good! Just a couple of grammar tweaks.

File: working-groups/query-standardization/use-cases/common/single_type/metrics/Join_on_labels.yaml (hunk 2 above)
Comment (Olli73773) [reply working-groups/query-standardization/use-cases/common/single_type/metrics/Join_on_labels.yaml]: Fixed the grammar

**Review by manolama — APPROVED** [review]

**Review by alolita — APPROVED** [review]
+1

**Pull request merged**